sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/QueryPlanningTracker.scala

```diff
@@ -120,12 +131,29 @@ class QueryPlanningTracker {
     ret
   }
 
+  /**
+   * Set when the query has been analysed and is ready for execution. This is after analysis for
+   * eager commands and after planning for other queries. see @link
+   * org.apache.spark.sql.execution.CommandExecutionMode When called multiple times, ignores
+   * subsequent call
```

Comment:
nit: would be more readable with adjusted spacing
```suggestion
   * Set when the query has been analysed and is ready for execution. This is after analysis for
   * eager commands and after planning for other queries.
   * see @link org.apache.spark.sql.execution.CommandExecutionMode
   * When called multiple times, ignores subsequent call.
```